db_main.go

```diff
@@ -100,6 +102,40 @@ func (db *DumbDB) Get(key []byte, bucket string) (ret_val []byte, err error) {
 	return
 }
 
+/*
+ * GetMultiple
+ * GetMultiple values given multiple keys
+ * @param 	keys		[][]byte slice containing keys
+ * @param 	bucket		name of bucket
+ * @returns 	values		return values as slices of byte slice
+ */
+func (db *DumbDB) GetMultiple(keys [][]byte, bucket string) (values [][]byte, err error) {
+
+	err = db.dbP.View(func(tx *bolt.Tx) error {
+		bkt := tx.Bucket([]byte(bucket))
+		if bkt == nil {
+			db.err_log.Println("Bucket not created yet.")
+			return bolt.ErrBucketNotFound
+		}
+
+		for _, key := range keys {
+			value := bkt.Get(key)
+			if value != nil {
+				values = append(values, value)
+			} else {
+				db.err_log.Println("Could not find value for key:%s", key)
```

Comment:
set values = nil here

Wont make too much of a difference.... but its better to return nil if err is set